chain/beacon/beacon.go

```diff
@@ -110,10 +113,10 @@ func ValidateBlockValues(bSchedule Schedule, nv network.Version, h *types.BlockH
 }
 
 func BeaconEntriesForBlock(ctx context.Context, bSchedule Schedule, nv network.Version, epoch abi.ChainEpoch, parentEpoch abi.ChainEpoch, prev types.BeaconEntry) ([]types.BeaconEntry, error) {
-	// Before nv22 we had "chained" beacons, and so required two entries at a fork
-	if nv < network.Version22 {
+	// When we have "chained" beacons, two entries at a fork are required.
+	currBeacon := bSchedule.BeaconForEpoch(epoch)
+	if currBeacon.IsChained() {
```

Comment:
Can we simplify this to: 

```
		parentBeacon := bSchedule.BeaconForEpoch(parentEpoch)
		currBeacon := bSchedule.BeaconForEpoch(epoch)
		if parentBeacon != currBeacon && currBeacon.IsChained() {
// fork logic
}
```

Then we can also drop line 140 (and just use currBeacon everywhere).

Reply:
Fixed